ntoskrnl/include/internal/mm.h

```diff
@@ -1653,15 +1653,30 @@ MmCheckSystemImage(
 NTSTATUS
 NTAPI
 MmCallDllInitialize(
-    IN PLDR_DATA_TABLE_ENTRY LdrEntry,
-    IN PLIST_ENTRY ListHead
-);
+    _In_ PLDR_DATA_TABLE_ENTRY LdrEntry,
+    _In_ PLIST_ENTRY ModuleListHead);
 
 VOID
 NTAPI
 MmFreeDriverInitialization(
     IN PLDR_DATA_TABLE_ENTRY LdrEntry);
 
+/* ReactOS-only, used by psmgr.c LookupEntryPoint() as well */
+NTSTATUS
+NTAPI
+RtlpFindExportedRoutineByName(
+    _In_ PVOID DllBase,
+    _In_ PCSTR ExportName,
+    _Out_ PVOID* Function,
+    _In_opt_ BOOLEAN CheckIfForwarder);
```

Comment:
I don't think this can be an `opt` argument, considering it is not a pointer.

Reply:
And maybe something like `AllowForwarder` or `DisallowForwarder` is a better name,
`CheckIfForwarder` does not describe what is happening when it is a forwarder at all.

Reply:
This has been re-architectured.